Assets/HoloToolkit/UX/Scripts/Collections/ObjectCollection.cs

```diff
@@ -434,6 +448,27 @@ private Vector3 CylindricalMapping(Vector3 source, float radius)
             return newPos;
         }
 
+        /// <summary>
+        /// Internal function for getting the relative mapping based on a source Vec3 and a radius for cylinder mapping.
+        /// </summary>
+        /// <param name="source">The source <see cref="Vector3"/> to be mapped to cylinder</param>
+        /// <param name="radius">This is a <see cref="float"/> for the radius of the cylinder</param>
+        /// <param name="row">This is a <see cref="int"/> for the radius of the cylinder</param>
+        /// <param name="column">This is a <see cref="int"/> for the radius of the cylinder</param>
+        /// <returns></returns>
+        private Vector3 RadialMapping(Vector3 source, float radius, int row, int column)
```

Comment:
Do we not use the source?

Reply:
Why not just update source with new values instead of creating a new vector3 each time?

```
            Radius = radius >= 0 ? Radius : radius;
       
            source.x = 0f;
            source.y = 0f;
            source.z = (Radius/Rows) * row;

            float yAngle = _radialCellAngle * (column - (_columns * 0.5f)) + (_radialCellAngle * .5f);

            Quaternion rot = Quaternion.Euler(0.0f, yAngle, 0.0f);
            source = rot * source;

            return source;
```

Reply:
Also, why not just do pure Quaternion math so we get better results without gimbal lock?